frontend/components/common/formatters/AnimatedNumber.vue

```diff
@@ -43,7 +52,9 @@ export default Vue.extend({
     },
     methods: {
         format(value: number | BigNumber): string {
-            return formatToAutomaticDecimalPoints(value, this.decimalPlaces);
+            return formatToAutomaticDecimalPoints(value, this.decimalPlaces, {
```

Comment:
I then use the function again here to add the final decimal points and correctly format the numbers between the start and end value (when the animation plays)